README.md

```diff
@@ -1,2 +1,17 @@
-# mlkube
+# MLKube
 Machine Learning Infrastructure built on Kubernetes
+
+Kubernetes makes it possible for anyone to build container-based infrastructure.
+MLKube, an open source project, leverages the extensibility of Kubernetes to enable us to run machine learning infrastructure on top of Kubernetes.
```

Comment:
s/infrastructure/workload, IMO.

Reply:
I prefer `infrastructure` as this project is different from kubeflow :)

Reply:
Got you! :)